modules/test/playwright/helpers/json-web-services/JSONWebServicesAssetListEntryApiHelper.ts

```diff
@@ -69,7 +95,7 @@ export class JSONWebServicesAssetListEntryApiHelper {
 		);
 
 		return await this.apiHelpers.post(
-			`${liferayConfig.environment.baseUrl}${this.basePath}/update-asset-list-entry`,
+			`${liferayConfig.environment.baseUrl}${this.basePath}/update-asset-list-entry-type-settings`,
```

Comment:
modifying this url makes the test "Preview content in a widget page by segment" from modules/test/playwright/tests/layout-admin-web/main/simulationMenu.spec.ts is failing. I think we should either update this test accordingly or create a different post call.